projects/pluto_ng/system_constr.xdc

```diff
@@ -0,0 +1,201 @@
+set_property  -dict {PACKAGE_PIN B5   IOSTANDARD ANALOG} [get_ports s_vtt_ps_ddr4_sns_n]   ; ##  B5  IO_L21N_66_VTT_PS_DDR4_SNS_N
+set_property  -dict {PACKAGE_PIN A9   IOSTANDARD ANALOG} [get_ports s_1v2_ps_ddr4_sns_p]   ; ##  A9  IO_L22P_66_1V2_PS_DDR4_SNS_P
+set_property  -dict {PACKAGE_PIN A8   IOSTANDARD ANALOG} [get_ports s_1v2_ps_ddr4_sns_n]   ; ##  A8  IO_L22N_66_1V2_PS_DDR4_SNS_N
+
+set_property  -dict {PACKAGE_PIN G3   IOSTANDARD ANALOG} [get_ports s_0v85_mgtravcc_sns_p] ; ##  G3  IO_L4P_66_0V85_MGTRAVCC_SNS_P
+set_property  -dict {PACKAGE_PIN F3   IOSTANDARD ANALOG} [get_ports s_0v85_mgtravcc_sns_n] ; ##  F3  IO_L4N_66_0V85_MGTRAVCC_SNS_N
+set_property  -dict {PACKAGE_PIN H4   IOSTANDARD ANALOG} [get_ports s_5v0_sns_p]           ; ##  H4  IO_L6P_66_5V0_SNS_P
+set_property  -dict {PACKAGE_PIN H3   IOSTANDARD ANALOG} [get_ports s_5v0_sns_n]           ; ##  H3  IO_L6N_66_5V0_SNS_N
+set_property  -dict {PACKAGE_PIN A3   IOSTANDARD ANALOG} [get_ports s_1v2_sns_p]           ; ##  A3  IO_L8P_66_1V2_SNS_P
+set_property  -dict {PACKAGE_PIN A2   IOSTANDARD ANALOG} [get_ports s_1v2_sns_n]           ; ##  A2  IO_L8N_66_1V2_SNS_N
+set_property  -dict {PACKAGE_PIN B3   IOSTANDARD ANALOG} [get_ports s_1v8_mgtravtt_sns_p]  ; ##  B3  IO_L10P_66_1V8_MGTRAVTT_SNS_P
+set_property  -dict {PACKAGE_PIN B2   IOSTANDARD ANALOG} [get_ports s_1v8_mgtravtt_sns_n]  ; ##  B2  IO_L10N_66_1V8_MGTRAVTT_SNS_N
+
+# clocks
+
+create_clock -name spi0_clk       -period  100   [get_pins -hier */EMIOSPI0SCLKO]
+
+create_clock -name ref_clk        -period  8.00  [get_ports fpga_ref_clk_p]
+
+create_clock -name rx1_dclk_out   -period  2.034 [get_ports rx1_dclk_in_p]
+create_clock -name rx2_dclk_out   -period  2.034 [get_ports rx2_dclk_in_p]
+create_clock -name tx1_dclk_out   -period  2.034 [get_ports tx1_dclk_in_p]
+create_clock -name tx2_dclk_out   -period  2.034 [get_ports tx2_dclk_in_p]
+
+# Allow max skew of 0.25 ns between input clocks
+set_clock_latency -source -early -0.125 [get_clocks rx1_dclk_out]
+set_clock_latency -source -early -0.125 [get_clocks rx2_dclk_out]
+
+set_clock_latency -source -late 0.125 [get_clocks rx1_dclk_out]
+set_clock_latency -source -late 0.125 [get_clocks rx2_dclk_out]
+
+set_property CLOCK_DELAY_GROUP BALANCE_CLOCKS_1 \
+  [list [get_nets -of [get_pins i_system_wrapper/system_i/axi_adrv9001/inst/i_if/i_rx_1_phy/i_div_clk_buf/O]] \
+        [get_nets -of [get_pins i_system_wrapper/system_i/axi_adrv9001/inst/i_if/i_rx_1_phy/i_clk_buf_fast/O]] \
+  ]
+
+set_property CLOCK_DELAY_GROUP BALANCE_CLOCKS_2 \
+  [list [get_nets -of [get_pins i_system_wrapper/system_i/axi_adrv9001/inst/i_if/i_rx_2_phy/i_div_clk_buf/O]] \
+        [get_nets -of [get_pins i_system_wrapper/system_i/axi_adrv9001/inst/i_if/i_rx_2_phy/i_clk_buf_fast/O]] \
+  ]
+
+ # MCS is synchronous fpga_ref_clk
+
+ set_input_delay -clock [get_clocks fpga_ref_clk] [get_property PERIOD [get_clocks fpga_ref_clk_p]] \
```

Comment:
I am getting the following warnings from this line:

WARNING: [Vivado 12-627] No clocks matched 'fpga_ref_clk'. [/export/data/workspace/ws6/hdl/proj
ects/pluto_ng/system_constr.xdc:200]
INFO: [Vivado 12-626] No clocks found. Please use 'create_clock' or 'create_generated_clock' co
mmand to create clocks. [/export/data/workspace/ws6/hdl/projects/pluto_ng/system_constr.xdc:200
]
WARNING: [Vivado 12-627] No clocks matched 'fpga_ref_clk_p'. [/export/data/workspace/ws6/hdl/pr
ojects/pluto_ng/system_constr.xdc:200]
INFO: [Vivado 12-626] No clocks found. Please use 'create_clock' or 'create_generated_clock' co
mmand to create clocks. [/export/data/workspace/ws6/hdl/projects/pluto_ng/system_constr.xdc:200
]
CRITICAL WARNING: [Common 17-55] 'get_property' expects at least one object. [/export/data/workspace/ws6/hdl/projects/pluto_ng/system_constr.xdc:200]
Resolution: If [get_<value>] was used to populate the object, check to make sure this command returns at least one valid object.

Reply:
Done